servers/zms/src/main/java/com/yahoo/athenz/zms/store/file/FileConnection.java

```diff
@@ -1620,4 +1620,47 @@ public boolean confirmRoleMember(String domainName, String roleName, RoleMember
         }
         return domainRoleMembersMap;
     }
+
+    @Override
+    public Set<String> getPendingMembershipApproverRoles() {
+
+        String[] fnames = getDomainList();
+        Set<String> roleNames = new HashSet<>();
+        List<String> slist = new ArrayList<>(java.util.Arrays.asList(fnames));
```

Comment:
Isn't java.util.Arrays.asList(fnames) already give you list? Why calling ArrayList with it?